<!DOCTYPE html>
<html>
<head>
  <title>pirovc</title>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Generator" content="CV Maker - https://cvmkr.com/" />
  
  <script type="text/javascript" src="https://cdn.jsdelivr.net/gh/pcooksey/bibtex-js/src/bibtex_js.js"></script>
  <bibtex src="https://raw.githubusercontent.com/pirovc/pirovc.github.io/main/pirovc.bib"></bibtex>
  <script type='text/javascript'>
    function loaded() {
      setTimeout(function(){
        
        // load scripts 1 sec. after page loaded to get DOIs from bibtex-js
        // ALTMETRICS (could add directly and call _altmetric_embed_init();)
        var script = document.createElement('script');
        script.src = "https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js";
        document.head.appendChild(script);
        
        // DIMENSIONS citations
        var script = document.createElement('script');
        script.src = "https://badge.dimensions.ai/badge.js";
        document.head.appendChild(script);
        
      },1000);
    }

  </script>
  <style type="text/css">
    body {
      background: #f0f0f0;
      margin: 0;
      padding: 0;
      
      font-family: Georgia, serif, 'Trebuchet MS';
      line-height: 20px;
      font-size: 15px;

      color: #222;
    }
    a:visited,a:link,a:active {
        color: #006400;
    }
    a:hover {
        color: #006400;
        font-weight: bold;
    }

    h1, h2, h3, h4 {
      margin: 0 0 15px 0;
      padding: 0;
    }
    h1 {
      font-size: 300%;
    }
    h2 {
      font-size: 110%;
    }
    h3 {
      font-size: 120%;
    }
    
    p {
      margin: 0 0 10px 0;
    }
    
    ul {
      list-style-type: circle;
    }
    li {
      margin-bottom: 5px;
    }
    a {
      color: #222;
    }
    /* ________________________________________________________ */
    .rtl {
      direction: rtl;
    }
    .clear {
      clear: both;
    }

    #main {
      background: #fff;
      width: 900px;
      padding: 35px;
      margin: 0 auto 15px auto;
      overflow: hidden;
      
      box-shadow: 0 0 6px #ccc;
    }
    
    #toptitle {
      line-height: 42px;
      margin: 0px 0 15px 0;
      font-weight: normal;
      text-align: center;
    }
    
    #credit {
      margin-bottom: 20px;
      font-size: 11px;
      color: #999;
      text-align: center;
    }
      #credit a {
        color: #999;
      }
    
    p label {
      font-weight: bold;
      
      font-size: 110%;
      margin-right: 10px;
    }
      p label span.colon {
        display: inline;
      }

    
    .section {
      margin-bottom: 30px;
    }
    
    .item {
      margin: 0 0 20px 0;
    }

    /*  _____________________________ */
    .basic-info {
      margin: 0 auto 30px auto;
      width: 80%;
      overflow: hidden;
      text-align: center;
    }
    .basic-info p {
      margin: 0;
    }
    .basic-info .dot {
      display: inline;
      font-size: 14px;
      margin: 0 10px;
    }
    
    .section .subtitle {
      text-transform: lower;
      font-size: 150%;
      text-align: center;
      border-bottom: 1px solid #ccc;
      padding-bottom: 10px;
    }
      .section .set h3 {
        margin: 0;
      }
    .section span.value {
      display: block;
    }
    .section h3 span.l, .section span.r {
      width: 49%;
      float: left;
    }
      .section span.r {
        float: right;
        text-align: right;
      }
      .section-education .school,
      .section-work .job_title {
        font-size: 100%;
        display: block;
        margin: 5px 0 5px 0;
        font-style: italic;
      }

    /* ___ photo ___ */
    #photo {
      width: 160px;
      height: 200px;
      overflow: hidden;
      float: left;
      margin-top: 8px;
      border-radius: 4px;
    }
    #photo img {
      width: 100%;
      height: auto;
    }
    .header.photo {
      margin-left: 190px;
    }
    .header.photo #title {
      text-align: left;
    }
    .header.photo .basic-info {
      float: none;
      padding: 0;
      width: auto;
      text-align: left;
    }

    /* Create three unequal columns that floats next to each other */
    .column {
      float: left;
      padding: 10px;
    }

    .left {
      width: 78%;
    }

    .middle, .right {
      width: 7%;
    }

    .row:after {
      content: "";
      display: table;
      clear: both;
    }

    @media screen and (max-width: 950px) {
      body {
        background: #fff;
      }
      #main {
        box-shadow: none;
        padding: 0;
        padding: 5px;
      }
    }
  </style>
</head>
<body class="lang-en" onload="loaded();" >

<div id="main">
<div class="header">
  <h1 id="toptitle">Vitor C. Piro</h1>
  <div class="basic-info section">
    <div class="set">
      <p class="email">
          <a href="https://github.com/pirovc">github</a>
          <a href="https://www.researchgate.net/profile/Vitor_Piro">researchgate</a>
          <a href="https://genomic.social/@pirovc">mastodon</a>
          <br />
      </p>
      <div class="clear"> </div>
    </div>
    <div class="clear"> </div>
  </div>
</div>
<div class="clear"> </div>
<div class="section section-software">
  <h2 class="subtitle">Software</h2>
  <div class="set">
    <div class="item">
      <span class="info">
		<span class="dot">&sect;</span> <a href="https://github.com/pirovc/ganon">ganon2</a> is a k-mer-based classification tool that uses hierarchical interleaved Bloom filters to efficiently classify genomic sequences against large reference sets. It offers integrated database download and updates (RefSeq/GenBank), taxonomic profiling (NCBI/GTDB), binning, hierarchical classification and customised reporting.
        <br />
        <a href="http://bioconda.github.io/recipes/ganon/README.html"><img src="https://img.shields.io/badge/install%20with-bioconda-brightgreen.svg?style=flat-square" alt="" /></a> <a href="https://anaconda.org/bioconda/ganon"><img src="https://anaconda.org/bioconda/ganon/badges/downloads.svg" alt="" /></a> <img alt="GitHub Repo stars" src="https://img.shields.io/github/stars/pirovc/ganon?style=social" />
        <br />
        <br />
        <span class="dot">&sect;</span> <a href="https://github.com/pirovc/genome_updater">genome_updater</a> is a bash script that downloads and updates (non-redundant) snapshots of the NCBI Genomes repository (RefSeq/GenBank) with advanced filters, detailed logs and reports, file integrity checks, NCBI taxonomy and GTDB integration and support for parallel downloads.
        <br />
        <a href="http://bioconda.github.io/recipes/genome_updater/README.html"><img src="https://img.shields.io/badge/install%20with-bioconda-brightgreen.svg?style=flat-square" alt="" /></a> <a href="https://anaconda.org/bioconda/genome_updater"><img src="https://anaconda.org/bioconda/genome_updater/badges/downloads.svg" alt="" /></a> <img alt="GitHub Repo stars" src="https://img.shields.io/github/stars/pirovc/genome_updater?style=social" />
        <br />
        <br />
        <span class="dot">&sect;</span> <a href="https://github.com/pirovc/multitax">MultiTax</a>  is a Python package that provides a standardised set of functions for downloading, parsing, filtering and exploring multiple taxonomies, including GTDB, NCBI, Silva, Greengenes and Open Tree Taxonomy, as well as custom-formatted taxonomies.
        <br />
        <a href="http://bioconda.github.io/recipes/multitax/README.html"><img src="https://img.shields.io/badge/install%20with-bioconda-brightgreen.svg?style=flat-square" alt="" /></a> <a href="https://anaconda.org/bioconda/multitax"><img src="https://anaconda.org/bioconda/multitax/badges/downloads.svg" alt="" /></a> <img alt="GitHub Repo stars" src="https://img.shields.io/github/stars/pirovc/multitax?style=social" />
        <br />
        <br />
        <span class="dot">&sect;</span> <a href="https://github.com/pirovc/grimer">GRIMER</a> performs analysis of microbiome data and generates a portable and interactive dashboard integrating annotation, taxonomy and metadata with focus on contamination detection.
        <br />
        <a href="http://bioconda.github.io/recipes/grimer/README.html"><img src="https://img.shields.io/badge/install%20with-bioconda-brightgreen.svg?style=flat-square" alt="" /></a> <a href="https://anaconda.org/bioconda/grimer"><img src="https://anaconda.org/bioconda/grimer/badges/downloads.svg" alt="" /></a> <img alt="GitHub Repo stars" src="https://img.shields.io/github/stars/pirovc/grimer?style=social" />
        <br />
        <br />
        <span class="dot">&sect;</span> <a href="https://github.com/pirovc/taxsbp">TaxSBP</a> is am implementation of the approximation algorithm for the hierarchically structured bin packing problem based on the NCBI Taxonomy database.
        <br />
        <a href="http://bioconda.github.io/recipes/taxsbp/README.html"><img src="https://img.shields.io/badge/install%20with-bioconda-brightgreen.svg?style=flat-square" alt="" /></a> <a href="https://anaconda.org/bioconda/taxsbp"><img src="https://anaconda.org/bioconda/taxsbp/badges/downloads.svg" alt="" /></a> <img alt="GitHub Repo stars" src="https://img.shields.io/github/stars/pirovc/taxsbp?style=social" />
        <br />
        <br />
        <span class="dot">&sect;</span> <a href="https://github.com/pirovc/metameta">MetaMeta</a> is a pipeline to execute and integrate results from metagenome analysis tools. It provides an easy workflow to run multiple tools with multiple samples, producing a single enhanced output profile for each sample.
        <br />
        <a href="http://bioconda.github.io/recipes/metameta/README.html"><img src="https://img.shields.io/badge/install%20with-bioconda-brightgreen.svg?style=flat-square" alt="" /></a> <a href="https://anaconda.org/bioconda/metameta"><img src="https://anaconda.org/bioconda/metameta/badges/downloads.svg" alt="" /></a> <img alt="GitHub Repo stars" src="https://img.shields.io/github/stars/pirovc/metameta?style=social" />
        <br />
        <br />
        <span class="dot">&sect;</span> <a href="https://github.com/pirovc/dudes">DUDes</a> is a reference-based taxonomic profiler with a top-down approach to analyze metagenomic NGS samples. Instead of using the lowest common ancestor we developed the deepest uncommon descendent.
        <br />
        <a href="http://bioconda.github.io/recipes/dudes/README.html"><img src="https://img.shields.io/badge/install%20with-bioconda-brightgreen.svg?style=flat-square" alt="" /></a> <a href="https://anaconda.org/bioconda/dudes"><img src="https://anaconda.org/bioconda/dudes/badges/downloads.svg" alt="" /></a> <img alt="GitHub Repo stars" src="https://img.shields.io/github/stars/pirovc/dudes?style=social" />
        <br />
        <br />
        <span class="dot">&sect;</span> <a href="https://github.com/pirovc/fgap">FGAP</a> is an automated gap closing tool. It uses BLAST to align multiple contigs against a draft genome assembly aiming to find sequences that overlap gaps. The algorithm selects the best sequence to fill and eliminate the gap.
        <br />
        <a href="http://bioconda.github.io/recipes/fgap/README.html"><img src="https://img.shields.io/badge/install%20with-bioconda-brightgreen.svg?style=flat-square" alt="" /></a> <a href="https://anaconda.org/bioconda/fgap"><img src="https://anaconda.org/bioconda/fgap/badges/downloads.svg" alt="" /></a> <img alt="GitHub Repo stars" src="https://img.shields.io/github/stars/pirovc/fgap?style=social" />
      </span>
    </div>
  </div>
  <div class="clear"> </div>
</div>


<div class="bibtex_template row">
    <div class="column left">
      <span class="title" style="font-weight: bold;font-size: 16px;"></span>
      <br>
      <span class="author"></span>
      <span class="journaltitle" style="font-style: italic;"></span>
      <span class="publisher" style="font-style: italic;"></span>
      <span class="date"></span>
      <a class="bibtexVar" href="http://dx.doi.org/+DOI+" extra="doi">
        https://dx.doi.org/<span class="doi"></span>
      </a>
    </div>
    <div class="column middle">
      <span class="__dimensions_badge_embed__ bibtexVar" data-legend="never" data-style="small_circle" data-doi="+DOI+" extra="doi"></span>
    </div>
    <div class="column right">
      <div class="altmetric-embed bibtexVar" data-badge-type="donut" data-doi='+DOI+' extra="doi"></div>
    </div>
  <br><br>
</div>

<div class="section section-publications">

<div class="bibtex_structure">
  <div class="sections bibtextypekey">
    
    <div class="section article">
        <h2 class="subtitle">Peer-reviewed publications</h2>
        <div class="clear"> </div>
        <div class="sort date" extra="DESC number">
          <h2 class="title"></h2>
          <div class="templates"></div>
        </div>
    </div>

    <div class="section misc">
        <h2 class="subtitle">Pre-prints</h2>
        <div class="clear"> </div>
        <div class="sort date" extra="DESC number">
          <h2 class="title"></h2>
          <div class="templates"></div>
        </div>
    </div>
  </div>
</div>


</div>



<div id="bibtex_display"></div>
<div id="credit">Webpage template from <a href="https://cvmkr.com/">CV Maker</a> | Publication list generated automatically with <a href="https://github.com/pcooksey/bibtex-js">bibtex-js</a></div>
</body>

</html>
